ci-operator/config/redhat-openshift-ecosystem/opct/.config.prowgen

```diff
@@ -0,0 +1,21 @@
+slack_reporter:
+- channel: "#forum-opct-updates"
+  job_states_to_report:
+  - failure
+  - error
+  report_template: |
+    :red_jenkins_circle: :workflow::external-link::vsphere: Periodical job *{{.Spec.Job}}* ended with *{{.Status.State}}*. (<{{.Status.URL}}|View logs>) (<https://prow.ci.openshift.org/job-history/gs/test-platform-results/logs/{{.Spec.Job}}|View history>)
+  job_names: # Listing of job names (ci-operator's 'as' field) that this configuration applies to
+  - platform-external-vsphere
+  - platform-external-vsphere-upgrade
+  - platform-none-vsphere
+  - platform-none-vsphere-upgrade
+  excluded_variants:
```

Comment:
You can probably remove this exluded_variants section for opct.  I am not seeing any okd variants